TheFourFifths/consus
# Add David dependencies and devDependencies badges

### Summary

This is a trivial change which adds a David badge for our npm dependencies and devDependencies.

### Checklist


Finally, is it ready to be reviewed and merged: yes :white_check_mark:

### How to Review

1. View [the readme-improvements branch](https://github.com/TheFourFifths/consus/tree/readme-improvements#consus)
2. Witness the stunning David badges
3. Click the badges, which should bring you to a list of our dependencies and devDependencies on david-dm.org

### Links

Put links to Jira tickets, and anything else, here.

- https://david-dm.org/
- https://david-dm.org/TheFourFifths/consus
- https://david-dm.org/TheFourFifths/consus?type=dev
